### PR TITLE
[Release fix] Stage rename

### PR DESCRIPTION
**Pull request by morsecodist**
# Description

Renames the stage in the db with a migration

# Tests

Ran locally and looked at my local runs from before the migration and their visualizations worked.

### CODE REVIEW

File: app/services/sfn_pipeline_dispatch_service.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ class SfnPipelineDispatchService
   # Constains SFN deployment stage names that differ from Rails.env
   ENV_TO_DEPLOYMENT_STAGE_NAMES = {
     "development" => "dev",
+    "staging" => "staging",
```
Comment (gregdingle) [inline app/services/sfn_pipeline_dispatch_service.rb:15]: how is this related?

Comment (morsecodist) [reply app/services/sfn_pipeline_dispatch_service.rb]: sorry, forgot to rebase
